frontend/App.js

```diff
@@ -31,13 +31,23 @@ export default function App() {
         <div className='change'>
           <button onClick={() => {
             setUiPleaseWait(true);
-            callSmartContractFunction('Top of the Mornin!');
+            callSmartContractFunction('Top of the Mornin!').then(_ =>
```

Comment:
the callbacks are identical for both button clicks. please make a func on line 24 like:
```
const updateMessage => message => {
  setUiPleaseWait(true);
            callSmartContractFunction(message).then(() =>
              {
                viewBlockchainState()
                  .then(val => setValueFromBlockchain(val));
                setUiPleaseWait(false)
              });
}
```
and do `<button onClick={() => updateMessage('go team')}>`

Also - please don't use `_` as an empty parameter for arrow funcs, just do `() => {}`